index.html

```diff
@@ -198,158 +165,249 @@ <h2>Tonic</h2>
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img class="v-social-media" src="images/vec.png" alt="vector"
+              /></a>
             </li>
           </ul>
-        </form>
+          <button type="button" class="click-btn">Get my resume</button>
+        </section>
+        <section>
+          <ul class="lang">
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Languages</p>
+                <img src="images/arrows-down.png" alt="arrow_bottom" />
+              </div>
+              <ul class="languages">
+                <li>
+                  <img src="images/js.png" alt="JavaScript" />
+                  <a href="#">JavaScript</a>
+                </li>
+                <li>
+                  <img src="images/html..png" alt="CSS" />
+                  <a href="#">HTML</a>
+                </li>
+                <li>
+                  <img src="images/css..png" alt="CSS" />
+                  <a href="#">CSS</a>
+                </li>
+              </ul>
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Frameworks</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Skills</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+          </ul>
+        </section>
       </div>
+    </section>
+    <footer>
+      <section class="contact">
+        <div class="contact-content">
+          <h1 class="contact-me">Contact me</h1>
+          <p class="paragraph item">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form
+          class="form-section"
+          action="https://formspree.io/f/mqknjqvv"
+          method="post"
+        >
+          <input
+            id="name"
+            class="text-holder1"
+            type="text"
+            name="name"
+            maxlength="30"
+            required
+            placeholder="Name"
+          />
+          <input
+            id="email"
+            class="text-holder1"
+            type="email"
+            name="name"
+            required
+            placeholder="Email"
+          />
+          <textarea
+            id="message"
+            class="text-area"
+            placeholder="Write your message here"
+            maxlength="500"
+            required
+          ></textarea>
+          <div id="form-button">
+            <button
+              id="clicked-btn"
+              class="click-btn"
+              type="submit"
+              name="sumit"
+            >
```

Comment:
- please fix the button on your contact form as correctly if Is not having the proper padding and font-size, please fix it so it looks like that of the Figma design.
- Please add the background-images as it is in the Figma design, to make your webpage more professional. 
- Please use the proper background color as it is in the Figma design `background: #6070FF;`

##### Your design
![Screenshot from 2022-07-16 10-07-56](https://user-images.githubusercontent.com/58771507/179348502-5ab3cc83-c161-48b5-83e8-7ad850d27350.png)


##### Figma design
![Screenshot from 2022-07-16 10-09-08](https://user-images.githubusercontent.com/58771507/179348546-67955ca6-8f38-4d43-a2dc-3ea01960c9b6.png)